foxpuppet/windows/browser/window.py

```diff
@@ -12,7 +12,8 @@
 
 class BrowserWindow(BaseWindow):
 
-    """Representation of a browser window."""
+    """Representation of a browser window.
```

Comment:
Hmm.. that's actually a good point. We probably don't want to document the constructor then! 😄